src/main/java/application/TodoResource.java

```diff
@@ -63,8 +64,8 @@ public Response getTodoById(@PathParam("todoId") @NotNull int todoId) {
   @Produces(MediaType.TEXT_PLAIN)
   public Response addTodo(BaseTodoDTO baseTodoDTO) {
     LOG.info("Create new todo");
-    Todo todo = todoService.addTodo(baseTodoDTO);
-    String stringResponse = "/api/todos/" + todo.getId();
+    long todoId = todoService.addTodo(baseTodoDTO);
+    String stringResponse = "/api/todos/" + todoId;
```

Comment:
`stringResponse` finde ich unpassend. Schau mal in die Spezifikation zum "CREATED" Response Code, was das zurückgibt, und benenne es entsprechend.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201